app/_components/commodities/Form.tsx

```diff
@@ -191,19 +195,27 @@ const Form: React.FC<FormProps> = ({ onSubmitHandler, isLoading }) => {
           ) : (
             <FormLabel marginY="auto">Selling</FormLabel>
           )}
-          <Stack paddingX={8} spacing={8} direction="row" mt={8}>
-            <Select
-              variant="customSelect"
-              id="buying"
-              marginY={'auto'}
-              onChange={(e) => {
-                setIsBuying(e.target.value === 'buying');
-              }}
-            >
-              <option value="buying">Buy</option>
-              <option value="selling">Sell</option>
-            </Select>
-
+          <Text>I am looking to:</Text>
+          <Stack spacing={8} direction="row" mt={8} flexWrap="wrap">
```

Comment:
To fit in with the other options, we should probably add a margin.
```suggestion
          <Stack margin={8} spacing={8} direction="row" mt={8} flexWrap="wrap">
```

Reply:
Are these resolved?